front/app/containers/MobileNavbar/FullMobileNavMenu.tsx

```diff
@@ -127,13 +129,14 @@ const FullMobileNavMenu = ({
 }: Props & InjectedIntlProps) => {
   const navbarItems = useNavbarItems();
   const pageSlugById = usePageSlugById();
+  const localize = useLocalize();
 
   if (isNilOrError(navbarItems) || isNilOrError(pageSlugById)) return null;
 
   const navbarItemPropsArray = getNavbarItemPropsArray(
     navbarItems,
     pageSlugById
-  );
+  ).filter(({ linkTo }) => linkTo !== '/projects');
```

Comment:
Could you add a comment here (projects is already in the 'outer' fixed menu)? This won't be clear for the next person why we're doing this.

Reply:
@alexander-cit, do you want to take care of this? You can pull/push directly to this branch